### DISCUSSION TITLE
Picture Carousel

### DISCUSSION THREAD

**Discussion by git24chris** [imfing/hextra#375] [Ideas]
Dear all, 
This theme is wow to me, and I am an end-user/consumer who decided to use it for my blog. Great work keeping it up-to-date, also community-wise. Thanks a lot.

Could you please consider adding a picture carousel feature to this theme? 

This feature should allow me to define a picture selection that can be horizontally scrolled with an arrow to the left and right to be clicked by the reader. Each picture should have an optional individual headline and an optional individual describing text below it. The already ongoing discussion about the "click-to-zoom" feature would complement it.

I found some examples in other blogs and home pages but I don't have the skills/capabilities to adopt these solutions.

Might that be a feature request?
Any link to an easy to implement existing solution is also very appreciated.

BR
  Chris

_User Story:_ 
As a blog author, I would like to offer a picture carousel based on a set of pictures with an optional header and footer for each picture, in order to invite the viewer to horizontally scroll left and right through an illustrated picture collection.

**Comment by imfing** [comment]
Hi @git24chris 

This is not currently offered out-of-the-box. 
I'd recommend google it and see how others implement such feature. For example https://hugocodex.org/add-ons/slider-carousel/

**Reply by git24chris** [reply]
Hi @imfing 

Thanks for your quick reply and for pointing me to that solution. That's exactly what I had in mind. While I failed to adopt other solutions I found online, this one has a crystal-clear description that works for me. Supercool.

Thanks a lot, and thumbs up!
